yew-macro/tests/macro/html-component-fail.stderr

```diff
@@ -1,186 +1,214 @@
 error: this opening tag has no corresponding closing tag
-  --> $DIR/html-component-fail.rs:79:13
+  --> $DIR/html-component-fail.rs:78:13
    |
-79 |     html! { <Child> };
+78 |     html! { <Child> };
    |             ^^^^^^^
 
-error: expected identifier
-  --> $DIR/html-component-fail.rs:80:22
+error: unexpected end of input, expected identifier
+  --> $DIR/html-component-fail.rs:79:13
    |
-80 |     html! { <Child:: /> };
-   |                      ^
+79 |     html! { <Child:: /> };
+   |             ^^^^^^^^^^^
```

Comment:
At the time of writing this is the only *expected* error span regression.
Instead of boring you with *why* this happens (TL;DR: blame `syn`) let me instead tell you about the silver lining:
This type of span error is already present right now, it can happen in props / attributes when there's an unexpected end of input (and also some other weird cases). When it happens the error span unfortunately points at the entire call site giving little to no helpful indication of the actual error location.

I used some hacks in this PR to make the span point at the offending tag instead.
It's currently impossible to get the exact location but pointing at the correct tag already helps a ton. Especially in large `html!` trees.